_posts/2016-08-05-rgsoc-team-rookies-month-1.md

```diff
@@ -0,0 +1,42 @@
+
+Hey, we are [Tehetena](https://github.com/titay2) and [HyeSoo](https://github.com/flyjwayur) from team [Rookies](https://twitter.com/hoodierookies). We want to keep our beginner mindset throughout the RGSOC journey, so we made our team name ‘Rookies’. We’re living and working from Helsinki, Finland. We are studying information technology and we love to learn new technologies. We joined the Rails Girls Summer of Code and it has been an amazing experience. 
+
+We started working on the Hoodie desktop application project. For making it, we’ve been working on UI/UX for the application and learning how to use Git, Node, localStorage and APIs to implement functionalities. 
+[Gregor](https://twitter.com/gr2m) is our project mentor. [Esko](https://twitter.com/orfjackal), [Omid](https://twitter.com/omidfi) and [Musse](https://github.com/mussebekabil) are our coaches, and [Pilar](https://twitter.com/Althaire) is our project supervisor. 
+
+![Kick-off meeting with mentor, coaches](/blog/images/201608/team-Rookies-kickoffcall.jpg)
+<font color="grey"><small><i>Rookies team in the main, at the bottom-right corner, from the left, are Esko, Gregor, Omid and Pilar. (Image: Pilar Huidobro)</i></small></font>
+
+## What we’ve achieved so far
+When we started the project, we had low self confidence. Even though we have taken classes on different programming languages, when it came to working with it we thought we did not have all the knowledge it took to get the job done. Now we understand that __every working experience is a great opportunity to teach ourselves new things everyday__.
+
+The very first lesson we learned was __Git__. We have done several school projects that we have managed to save and clone project repositories on Git, but after we started working on the Hoodie desktop application, there have been many challenges. At first, we did not know how to propose issues, submit pull requests, fetch from remote, rebase and so on, and we were confused with the concept between local and remote. With our mentor Gregor’s help, and many trials and errors, we now have better understanding of how GitHub actually works, how it is used in our project, and also when many contributors are working on the same project.
+
+Other than using GitHub, we have also learned a lot on __UI building__ and how to implement functionalities on the application using HTML, CSS, and JavaScript. We learned how to use EventListener, event.target, .map, .forEach, and localStorage. Using localStorage and making an API were very interesting. We followed the [tutorials](https://github.com/gr2m/localstorage-tutorial/) made by Gregor. It was very helpful and easy to follow. After practicing the code with tutorials, we had a call to solve the problems together and had a Q&A session. We have a call almost everyday. It’s been really helpful and given us deeper insight on how coding can be applied in the application and how we can use it in the desktop application with the Electron framework.
```

Comment:
Newbies might not know what UI stands for. You can add the abbreviation like `UI (user interface)`.